magento/magento2
# [FIXES] #25674: Elasticsearch version selections in admin are overly …

…broad.





### Description (*)
This removes the X.0+ notation and replaces it with X.x notation


### Fixed Issues (if relevant)

1. magento/magento2#25674: Elasticsearch version selections in admin are overly broad

### Manual testing scenarios (*)

1. Login to Admin
2. Go to Stores -> Configuration -> Catalog -> Catalog
3. Open Catalog Search Configuration
4. Check the entries in the dropdown "Search Engine"

### Questions or comments


### Contribution checklist (*)
